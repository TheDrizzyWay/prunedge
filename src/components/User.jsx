import React from 'react';

const User = () => {
    return (
        <div className="user">
            <div className="user__image">
                <img src="" alt="profile"/>
            </div>
            <div className="user__fullname">
                <p class="user__fullname--bold">Jaohne Samue</p>
            </div>
            <div className="user__designation">
                <p>Front Desk Officer</p>
            </div>
            <div className="user__department">
                <p>Finance</p>
            </div>
      </div>
    );
}

export default User;
