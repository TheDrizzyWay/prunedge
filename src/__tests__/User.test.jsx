import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import User from '../components/DocumentsView';

configure({ adapter: new Adapter() });

describe('User component', () => {
  it('should render user component', () => {
    const wrapper = shallow(
      <User />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
