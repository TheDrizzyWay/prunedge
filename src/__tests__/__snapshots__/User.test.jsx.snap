// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User component should render user component 1`] = `ShallowWrapper {}`;
